frontend/src/components/AccountReceiveTable.vue

```diff
@@ -97,7 +97,7 @@
               <q-card-section>
                 <div class="row justify-between items-center">
                   <div>{{ $t('AccountReceiveTable.sender') }}</div>
-                  <div @click="copyAddress(props.row.from, 'Sender')" class="cursor-pointer copy-icon-parent">
+                  <div @click="copyAddress(props.row.from, provider, 'Sender')" class="cursor-pointer copy-icon-parent">
```

Comment:
_(arbitrary comment location to start a thread)_

> I like the link in the wallet area, only thing I would say is it might also be good to include a link to it on the send form itself for better discoverability

Reply:
Added link to sent page on the send page

Reply:
LGTM, might do them as two outline buttons side by side, but ok with doing that in a follow up PR to avoid churn/blocking this, as long as @apbendi approves